CONTRIBUTING.md

```diff
@@ -388,23 +388,30 @@ cd superset
 
 The latest documentation and tutorial are available at https://superset.apache.org/.
 
-The site is written using the Gatsby framework and docz for the
-documentation subsection. Find out more about it in `docs/README.md`
+The documentation site is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator, the source for which resides in `./docs`.
 
-#### Images
```

Comment:
Is the info about Images no longer relevant?

Reply:
It looks to me like that's a vestige from Gatsby/RST; from what I can tell images for the documentation work in [the default Docusaurus way](https://docusaurus.io/docs/static-assets), so you just put them in `docs/static/img/...` and reference them as `img/...`.